scripts/upgrade/upgrade-clients.ts

```diff
@@ -151,8 +151,19 @@ export async function upgradeClients(env: string, targetVersion: string, dryRun:
   );
   await fermionFNFT.waitForDeployment();
   const fermionFNFTAddress = await fermionFNFT.getAddress();
-  console.log(`✅ FermionFNFT deployed at: ${fermionFNFTAddress}`);
-
+  deploymentComplete(
+    "FermionFNFT",
+    fermionFNFTAddress,
+    [
+      bosonPriceDiscoveryAddress,
+      protocolAddress,
+      wrappedNativeAddress,
```

Comment:
```suggestion
      protocolAddress,
      dependencies.seaportWrapperAddress,
      strictAuthorizedTransferSecurityRegistry,
      wrappedNativeAddress,
```
Maybe put everything into a `contructorArgs` array to ensure the same values are used.